forge-gui-mobile/src/forge/Forge.java

```diff
@@ -901,6 +901,7 @@ public void render() {
                             currentScene.render();
                             currentScene.act(delta);
                         } catch (IllegalStateException | NullPointerException ie) {
+                            String x = "xyz".substring(2);
```

Comment:
debugging code?

Reply:
Debugging code. Needed to throw something in there to be able to hit a breakpoint and see those exceptions. Removed, thanks for catching it.